class/defaults.yml

```diff
@@ -16,3 +16,31 @@ parameters:
 
     helm_values:
       enableCertManager: true
+
+    monitoring_enabled: true
+
+    alerts:
+      PatchReconcileSlow:
+        enabled: true
+        rule:
+          annotations:
+            description: Patch operator controller {{$labels.controller}} has a high reconcile time for 10 minutes
+            message: Patch operator controller {{$labels.controller}} has a high reconcile time for 10 minutes
+            summary: Patch operator controller has a high reconcile time for 10 minutes
```

Comment:
We generally require that alerts managed by components have a runbook, cf. https://syn.tools/syn/explanations/commodore-components/alerts.html. We generally manage the runbooks as part of the component documentation, so we can link to them on https://hub.syn.tools.

See component rook-ceph for an example: 
* https://github.com/projectsyn/component-rook-ceph/blob/00b00bddd0c2381963f89c7f517a89821fa140ad/component/alertrules.libsonnet#L66-L67
* https://github.com/projectsyn/component-rook-ceph/blob/00b00bddd0c2381963f89c7f517a89821fa140ad/component/alertrules.libsonnet#L108 
* any of the runbooks in https://github.com/projectsyn/component-rook-ceph/blob/master/docs/modules/ROOT/pages/runbooks, for example https://github.com/projectsyn/component-rook-ceph/blob/master/docs/modules/ROOT/pages/runbooks/RookCephOperatorScaledDown.adoc
* navigation links need to be added in `docs/modules/ROOT/partial/nav.adoc